src/bridge/IrcBridge.ts

```diff
@@ -418,6 +424,7 @@ export class IrcBridge {
                     {server: server.domain},
                     mxMetrics["connection_failure_kicks"] || 0
                 );
+                maxRemoteGhosts.set({server: server.domain}, server.getMaxClients());
```

Comment:
We set this inside the collector because hot reloading means the value can change.